packages/uiweb/src/lib/components/chat/ChatProfile/AddWalletContent.tsx

```diff
@@ -45,61 +29,25 @@ export const AddWalletContent = ({
   memberList,
   handleMemberList,
   groupMembers,
-  isLoading,
-  title,
-  submitButtonTitle,
+  isLoading = false,
+  groupInfo,
 }: AddWalletContentProps) => {
-  const theme = useContext(ThemeContext);
 
-  const [filteredUserData, setFilteredUserData] = useState<any>(null);
-  const { account, env } = useChatData();
-  const isMobile = useMediaQuery(device.mobileL);
-  const {fetchChatProfile} = useGetChatProfile();
+  const { fetchMemberStatus } = useGroupMemberUtilities();
   const groupInfoToast = useToast();
-  const customSearchStyle:CustomStyleParamsType = {
-   background:theme.backgroundColor?.modalInputBackground,
-   border:theme.border?.modalInnerComponents,
-   placeholderColor:theme.textColor?.modalSubHeadingText,
-   fontSize:'15px',
-   fontWeight:'400'
-  };
-
-  const handleSearch = async ({searchedText}:{searchedText:string}): Promise<void> => {
-    //fix ens search 
-    const newChatUser = await getNewChatUser({
-      searchText: searchedText,
-      fetchChatProfile,
-      env,
-    });
-    if(newChatUser){
-      setFilteredUserData(newChatUser);
-    }
-    else{
-        groupInfoToast.showMessageToast({
-          toastTitle: 'Error',
-          toastMessage: 'Invalid Address',
-          toastType: 'ERROR',
-          getToastIcon: (size) => <MdError size={size} color="red" />,
-        });
-    }
-  };
-
-  const clearInput = () => {
-    setFilteredUserData(null);
-  };
-
-  const addMemberToList = (member: User) => {
+  
+  const addMemberToList = async (member: IUser) => {
     let errorMessage = '';
-    const isMemberAlreadyAdded = memberList?.find(
-      (user: any) => user.wallets.toLowerCase() === member.wallets.toLowerCase()
-    );
-    console.log('member', member);
-
+    const memberStatus = await fetchMemberStatus({
+      chatId: groupInfo!.chatId!,
+      accountId: member!.wallets,
+    });
     errorMessage = addWalletValidation(
       member,
       memberList,
       groupMembers,
-      account
+      memberStatus!,
+      groupInfo?.isPublic ? 25000 : 5000
```

Comment:
is it suppose to be 5000 for private groups and 25000 for public ?

Reply:
yes

Reply:
it's not working as expected then like if I'm trying to create a group then after selecting whether it is public or private we go to the add more wallets section so there it shows 25000 only irrespective of if we have chose public or private